src/fswAlgorithms/orbitControl/lambertSolver/_UnitTest/test_lambertSolver.py

```diff
@@ -0,0 +1,321 @@
+#
+#  ISC License
+#
+#  Copyright (c) 2023, Autonomous Vehicle Systems Lab, University of Colorado Boulder
+#
+#  Permission to use, copy, modify, and/or distribute this software for any
+#  purpose with or without fee is hereby granted, provided that the above
+#  copyright notice and this permission notice appear in all copies.
+#
+#  THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+#  WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+#  MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+#  ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+#  WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+#  ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+#  OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+#
+#
+import copy
+import itertools
+
+import pytest
+from Basilisk.architecture import messaging
+from Basilisk.fswAlgorithms import lambertSolver
+from Basilisk.utilities import SimulationBaseClass
+from Basilisk.utilities import macros
+from Basilisk.utilities import orbitalMotion
+
+from fswAlgorithms.orbitControl.lambertSolver._UnitTest.Support.IzzoLambert import *
+
+# parameters
+solver = ["Gooding", "Izzo"]
+revs = [0, 1, 4]
+times = [1e2, 1e6]
+eccentricities = [0.0, 0.05, 1.0, 1.2]
+transferAngle = [30., 90., 180., 210., -60., 500.]
+alignmentThreshold = [1.]
+
+paramArray = [solver, revs, times, eccentricities, transferAngle, alignmentThreshold]
+# create list with all combinations of parameters
+paramList = list(itertools.product(*paramArray))
+# transfer angles >= 180 or < 0 not applicable for parabolic and hyperbolic transfer. Delete from list.
+paramList = [item for item in paramList if not (item[3] >= 1.0 and (item[4] >= 180. or item[4] < 0.))]
+# add cases to check alignmentThreshold module parameter
+paramList.append((solver[0], revs[0], times[0], eccentricities[0], 0.5, 1.))
+paramList.append((solver[0], revs[0], times[0], eccentricities[0], 1.5, 1.))
+paramList.append((solver[0], revs[0], times[0], eccentricities[0], 1.5, 2.))
+paramList.append((solver[0], revs[0], times[0], eccentricities[0], 2.5, 2.))
+
+@pytest.mark.parametrize("accuracy", [1e-2])
+@pytest.mark.parametrize("p1_solver, p2_revs, p3_times, p4_eccs, p5_angles, p6_align", paramList)
+
+def test_lambertSolver(show_plots, p1_solver, p2_revs, p3_times, p4_eccs, p5_angles, p6_align, accuracy):
+    r"""
```

Comment:
The HTML documentation doesn't pull in this method definition?  this is because `python test_lamberSolver.py` returns an error message and thus can't be parsed.

Reply:
It runs fine on my computer and on CI, so I don't know where the issue lies. I rewrote this line to import IzzoLambert in a different way.